docs/src/pages/versions/versions.md

```diff
@@ -17,19 +17,38 @@ You can use it to see what changes are coming and provide better feedback to MUI
 
 ## Versioning strategy
 
-Stability ensures that reusable components and libraries, tutorials, tools, and learned practices don't become obsolete unexpectedly. Stability is essential for the ecosystem around MUI to thrive.
+Stability ensures that reusable components and libraries, tutorials, tools, and learned practices don't become obsolete unexpectedly.
+Stability is essential for the ecosystem around MUI to thrive.
 
-This document contains the practices that are followed to provide you with a leading-edge UI library, balanced with stability, ensuring that future changes are always introduced in a predictable way.
+This document contains the practices that are followed to provide you with a leading-edge UI library, balanced with stability, ensuring that future changes are always introduced predictably.
 
 MUI follows [Semantic Versioning 2.0.0](https://semver.org/).
 MUI version numbers have three parts: `major.minor.patch`.
 The version number is incremented based on the level of change included in the release.
 
-- **Major releases** contain significant new features, some but minimal developer assistance is expected during the update. When updating to a new major release, you may need to run update scripts, refactor code, run additional tests, and learn new APIs.
+- **Major releases** contain significant new features, some but minimal developer assistance is expected during the update.
+  These releases include [breaking changes](#what-doesnt-count-as-a-breaking-change).
```

Comment:
```suggestion
  These releases include breaking changes.
```

Why linking what doesn't count as breaking change on a "breaking change" link?

Reply:
The idea was to better define what the term means?